client/public/src/components/app.jsx

```diff
@@ -4,67 +4,89 @@ import axios from 'axios';
 
 import Details from './details.jsx';
 import Options from './options.jsx';
+import Store from './store.jsx';
 
 const App = () => {
-  const [productId, setProductId] = useState(1);
-  const [product, setProduct] = useState({});
-  const [storeId, setStoreId] = useState(1);
   const [stores, setStores] = useState([]);
   const [stock, setStock] = useState([]);
   const [colors, setColors] = useState([]);
   const [sizes, setSizes] = useState([]);
 
+  const [store, setStore] = useState({id: 1});
+  const [product, setProduct] = useState({id: 1});
+  const [color, setColor] = useState('red');
+  const [size, setSize] = useState('M');
+  const [qty, setQty] = useState(0);
+
   // get the products general details such as price, name and reviews
   const getProduct = async (productId) => {
     const response = await axios.get(`/products/${productId}`);
-    const productData = response.data[0];
+    const productData = response.data;
     setProduct(productData);
   }
 
+  const getStore = async (storeId) => {
+    const response = await axios.get(`/stores/${storeId}`);
+    const storeData = response.data;
+    setStore(storeData);
+  }
+
   // get a list of all the stores (names and ids)
-  const getStores = async () => {
+  const getstores = async () => {
```

Comment:
Do you want this to be getStores instead of getstores? If not, no problem. If you do change it, be sure to change it everywhere it's invoked.